javascript/reactjs-todo-davinci/.env.example

```diff
@@ -0,0 +1,7 @@
+API_URL=$API_URL
+DEBUGGER_OFF=true
+DEVELOPMENT=$DEVELOPMENT
+PORT=$PORT
+WEB_OAUTH_CLIENT=$WEB_OAUTH_CLIENT
+SCOPE="openid profile email phone name revoke"
```

Comment:
I am not sure we need the revoke scope anymore. Might be wrong on this for the JS SDK, but P1 clients now support revoke and session end using the IDToken. 
If this scope is not required for this embedded use case, we should remove as it is a non default scope

Reply:
I believe you are correct. Looks like my tests still pass when I remove it and I can see revoke being called successfully in the network tab